server/auth.js

```diff
@@ -21,13 +20,16 @@ passport.deserializeUser(
   }
 )
 
-// Local passport configuration
-
-auth.post('/local/login', (req, res, next) =>
-  passport.authenticate('local', {
-    successRedirect: '/'
+auth.post('/local/login', function (req, res, next) {
+  passport.authenticate('local', function (err, user, info) {
```

Comment:
So...what happens with this block right now, is that we're going to hit this `post` route, passport is going to read that we want to use the local strategy, it's going to go down to line 34 and execute that block. In the process, the username and password will be checked, and if they're good, then `req.user` will be set. _Then_ it will come back here and execute this callback function, which...logs the user in and sets `req.user` again.

My suggestion for this would be to set the `successRedirect` in this callback function to `/api/auth/whoami` which is on line 53, and then let the whoami route send `req.user` to the front. That way, we're not logging the person in and setting `req.user` twice...

Also, other small thing, can we use arrow functions whenever we can? If you don't like arrow functions, that's something we should decide on, so that we can be consistent throughout... (I can also set our linter to prefer one over the other.)